drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/clientview.rb

```diff
@@ -4,13 +4,22 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
+# To create a ClientView URL:
+# ::GrdaWarehouse::RemoteCredential.create!(
+# 	slug: 'ac_hmis_clientview',
+# 	endpoint: 'https://www.google.com', # URL base of ClientView
+# 	type: 'GrdaWarehouse::RemoteCredential',
+# 	username: '',
+# 	encrypted_password: '',
+# 	active: true,
+# )
```

Comment:
This is what I did locally to store a base URL for client view. Is this correct with howit's supposed to be used?  @eanders @ttoomey

Reply:
Good question. I was a little unclear on the clientview credential. It seems like it just exists to provide this one link_base variable but I'm probably missing something. Before `185000178-remote-credential-adjustments`,  what used to be there was:
```ruby
GrdaWarehouse::RemoteCredentials::ExternalLink.where(slug: 'clientview').first&.link_base
```

If you already had the clientview credential, it should have been [updated by this migration](https://github.com/greenriver/hmis-warehouse/pull/3009/files#diff-578626979af2debd8aff4977d01815b2ca6e9d22f777c92a8368dd46d50f5d67)

But to create it, I think what you have is fine, you just need to change the type to be ExternalLink (see suggestion above)

Reply:
I see, thanks. Yeah that's exactly what its for, just providing one URL value. Thanks for the suggestion, I figured I was missing a type. I'll update